src/sentry/api/serializers/models/group_stream.py

```diff
@@ -306,6 +317,8 @@ def serialize(
             }
             if "times_seen" in attrs:
                 result.update(self._convert_seen_stats(attrs))
+            if "is_unhandled" in attrs:
+                result["isUnhandled"] = attrs["is_unhandled"]
```

Comment:
Since this is a boolean would you ever want to have `isUnhandled: False` for the ones that are handled?

Reply:
yup, you would want `isUnhandled: False` for the ones that are handled. it shouldn't matter here though, because for the orgs without the feature flag, they will get the unhandled information from the main issues request. for those with the feature flag, they won't get it from the issues request and will get it here instead. so at the end, we will have the same data for the groups, but getting it from different places.